data-parser/schema/facebook/comment.go

```diff
@@ -44,3 +47,40 @@ func CommentArraySchemaLoader() *gojsonschema.Schema {
 	schema, _ := gojsonschema.NewSchema(schemaLoader)
 	return schema
 }
+
+type CommentORM struct {
+	gorm.Model
```

Comment:
`gorm.Model` is a basic GoLang struct which includes the following fields: ID, CreatedAt, UpdatedAt, DeletedAt.
We can build the model without it.